actors/eam/src/lib.rs

```diff
@@ -73,10 +74,15 @@ impl EthAddress {
         self.0[0] == 0xff && self.0[1..12].iter().all(|&i| i == 0)
     }
 
+    #[inline]
+    fn is_null(&self) -> bool {
```

Comment:
nit: is null a term used in the EVM? I think this is just the "system" address.

Reply:
I've seen it pop up in quite a bit while looking into the topic but in EVM itself I don't think it is called anything.
Try searching "Eth system address" vs "Eth null address" and you get more informational results with the former. Though really it doesn't matter to me so I am happy to change if you feel we should be self consistient since "system" address follows the naming scheme that we have.

Reply:
Fair enough. And yeah, it looks like etherscan calls this null.